src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  letter-spacing: 0;
+}
+
+nav > .container {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+a {
+  text-decoration: none;
+}
+
+.nav__link {
+  color: #000;
+}
+
+.apple {
+  color: #00acdc;
+}
+
+.laptops {
+  color: #00acdc;
+}
+
+.list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.logo {
+  margin-top: 10px;
+  margin-left: 50px;
+}
+
+.elements {
+  margin-right: 20px;
+  color: black;
+  position: relative;
+}
+
+ul:not(:first-child) {
+  margin-right: 50px;
+}
+
+.active::after {
+  content: "";
+  display: block;
+  position: absolute;
+  width: 37px;
+  height: 4px;
+  top: 26.5px;
+  border-radius: 8px;
+  background: #00acdc;
```

Comment:
These styles should be on active link (`<a>` tag), in your HTML `.active` is a class on `<li>`